contracts/root/staking/CustomSupernetManager.sol

```diff
@@ -27,6 +28,8 @@ contract CustomSupernetManager is ICustomSupernetManager, Ownable2StepUpgradeabl
 
     GenesisSet private _genesis;
     mapping(address => Validator) public validators;
+    IRootERC20Predicate private _rootERC20Predicate;
+    mapping(address => uint256) private _genesisBalances;
```

Comment:
We can expose the variable (make it `public`), and can remove the getter function below.

Reply:
Fixed https://github.com/0xPolygon/core-contracts/commit/485e3c6b13bba23a9e31ae63430c082981146177